src/simulator_worker/utils.py

```diff
@@ -105,10 +119,17 @@ def create_output_esdl(
     profiles = ProfileManager()
     profiles.profile_type = "DATETIME_LIST"
     profiles.profile_header = ["datetime"]
+
     for series_name, _ in simulation_result.items():
         logger.debug(f"Output series: {series_name}")
-        # print(f"{series_name}:\t\t {asset.port}")
-        # print(f"Inport index={get_port_index(asset, esdl.InPort)}")
+        asset = _id_to_asset(series_name[0], esh.energy_system)
+        port_index = -1
+        if series_name[1].endswith("supply"):
```

Comment:
Perhaps add a `series_name[1].lower().endswith...` or is it guaranteed that the series_name[1] is always lower cased?

Reply:
probably guaranteed (convention comes from simulator-core team),  but added anyways